star-ui/src/components/PidTuningPanel.tsx

```
@@ -0,0 +1,101 @@
+                </div>
+
+                {/* PID Inputs */}
+                {[
+                    { label: 'Kp (Proportional)', value: kp, set: setKp },
+                    { label: 'Ki (Integral)', value: ki, set: setKi },
+                    { label: 'Kd (Derivative)', value: kd, set: setKd },
+                ].map(param => (
+                    <div key={param.label}>
+                        <label style={{ fontSize: '10px', color: 'rgba(255,255,255,0.35)', textTransform: 'uppercase', letterSpacing: '0.06em', display: 'block', marginBottom: '4px' }}>
+                            {param.label}
+                        </label>
```

Biome (2.4.4)

[error] A form label must be associated with an input.

(lint/a11y/noLabelWithoutControl)